vuecomponents/tool/src/main/frontend/src/components/dynamic-criterion.vue

```diff
@@ -0,0 +1,164 @@
+<template>
+  <div class="crit-background" aria-describedby="title">
```

Comment:
I can't find an element with id="title", should `aria-describedby="title"` be removed or point somewhere?